Assets/Scripts/ROS Scripts/ROS Connections v2/Sensors/LampSensor_ROSSensorConnection.cs

```diff
@@ -198,56 +198,6 @@ public string GetMessageType(string topic)
         **/
     }
 
-    /// <summary>
```

Comment:
Check to see if this breaks the visualizer's coloring scheme.

Reply:
Any update on this?

Reply:
Or guidance on how we can check it?